runtime-k3d/src/runtime/config/helm/values.yaml

```diff
@@ -1,62 +1,47 @@
-# Copyright (c) 2022 Robert Bosch GmbH and Microsoft Corporation
-#
-# This program and the accompanying materials are made available under the
-# terms of the Apache License, Version 2.0 which is available at
-# https://www.apache.org/licenses/LICENSE-2.0.
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
-# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
-# License for the specific language governing permissions and limitations
-# under the License.
-#
-# SPDX-License-Identifier: Apache-2.0
-
-imageSeatService:
-  name: seatservice
-  repository: k3d-registry.localhost:12345/seatservice
-  pullPolicy: Always
-  daprAppid: seatservice
-  daprAppPort: 50051
-  daprLogLevel: debug
+imageMosquitto:
+  args:
+  - mosquitto
+  - -c
+  - /mosquitto-no-auth.conf
+  name: mqtt-broker
+  nodePort: '{{.Values.imageMosquitto.nodePort}}'
+  nodePort1: '{{.Values.imageMosquitto.nodePort1}}'
```

Comment:
For what is "nodePort1" and "port1" etc used for? can it be removed?

Reply:
No, it can't be removed, I will explain you in the call